app/components/candidate_interface/continuous_applications/application_summary_component.rb

```diff
@@ -73,6 +77,97 @@ def application_choice_status_row
+        return nil unless application_choice.offer_withdrawn?
+
+        if application_choice.offer_withdrawal_reason.present?
+          {
+            key: 'Reason for offer withdrawal',
+            value: application_choice.offer_withdrawal_reason,
+          }
+        end
+      end
+
+      def rejection_reasons_row(application_choice)
+        return unless application_choice.rejected?
+        return unless application_choice.rejection_reason.present? || application_choice.structured_rejection_reasons.present?
+
+        {
+          key: 'Feedback',
+          value: render(
+            RejectionsComponent.new(
+              application_choice:,
+              render_link_to_find_when_rejected_on_qualifications: @render_link_to_find_when_rejected_on_qualifications,
+              rejection_reasons_component: CandidateInterface::RejectionReasons::RejectionReasonsComponent,
+              feedback_button: true,
+            ),
+          ),
+        }
+      end
+
+      def successful_application?(application_choice)
+        application_choice.pending_conditions? || application_choice.offer? || application_choice.offer_deferred?
+      end
+
+      def ske_conditions_row(application_choice)
+        return unless successful_application?(application_choice)
+        return if (ske_conditions = application_choice.offer&.ske_conditions).blank?
+
+        {
+          key: 'Subject knowledge enhancement course'.pluralize(ske_conditions.size),
+          value: render(OfferSkeConditionsReviewComponent.new(ske_conditions:)),
+        }
+      end
+
+      def reference_conditions_row(application_choice)
+        return unless successful_application?(application_choice)
+        return if (reference_condition = application_choice.offer&.reference_condition).blank?
+
+        {
+          key: 'References',
+          value: render(OfferReferenceConditionReviewComponent.new(reference_condition:)),
+        }
+      end
+
+      def conditions_row(application_choice)
+        return unless successful_application?(application_choice)
+        return unconditional_offer_row if application_choice.unconditional_offer?
+
+        {
+          key: 'Condition'.pluralize(application_choice.offer.conditions.count),
+          value: render(
+            OfferConditionsReviewComponent.new(
+              conditions: application_choice.offer.non_structured_conditions_text,
+              provider: application_choice.current_course.provider.name,
+              application_form: application_choice.application_form,
+            ),
+          ),
+        }
+      end
+
+      def respond_to_offer_row(application_choice)
+        return unless application_choice.offer?
+
+        {
+          key: ' ',
+          value: render(
+            CandidateInterface::CourseChoicesSummaryCardActionComponent.new(
+              action: :respond_to_offer,
+              application_choice:,
+            ),
+          ),
+        }
+      end
```

Comment:
I had to decouple from the Dashboard component. I might modularize or adding specs to this component. Any thoughts?